api/v1alpha1/openapi.yaml

```diff
@@ -3009,3 +3009,157 @@ components:
       - 'TemplateDiscriminatorKubernetesSec'
       - 'TemplateDiscriminatorInlineConfig'
       - 'TemplateDiscriminatorHttpConfig'
+    CertSigningRequestApproval:
+      type: object
+      properties:
+        labels:
```

Comment:
Unlike with EnrollmentRequests, CSRs shouldn't change labels.

Reply:
Hm yes, that makes sense.